### DISCUSSION TITLE
[Feature] Show person by age

### DISCUSSION THREAD

**Discussion by jwnacnud** [immich-app/immich#12244] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

You have already added the ability to specify the date of birth - amazing.  You already have great filters to search by.  Can I also search for photos of a particular person at a certain age (only for one person)?  

Example:
Person: Bob Smith
Age: 18

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by alextran1502** [comment]
I love this idea!

**Reply by jwnacnud** [reply]
I know that I can do it right now because we have a date range that we can filter by.  I could take the person's date of birth and add the number of years and set the range to show that year, but that's what a calculation is for.

**Reply by mertalev** [reply]
It would only really work for 1 person right? If you set an age for one person, then the others would be indirectly set as well. In terms of UI, it could be set up so that setting a person's age just automatically configures the date window as a shorthand.

**Reply by jwnacnud** [reply]
Yes, only one person at a time.  I modified the original feature request.
